### PR TITLE
add, restore and delete in trash

### PR DESCRIPTION
**Pull request by Jay-Gadhiya**
Test Link: https://cheer-notes-mh8r4s2cf-jay-gadhiya.vercel.app/

context : noteActions-context.jsx  reducer : noteActions-reducer.jsx

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ import { HomePage } from "./Pages/Home/Home";
 import { Login } from "./Pages/Authentication/Login";
 import { Signup } from "./Pages/Authentication/Signup";
 import { Archive } from "./Pages/Archive-Page/Archive";
+import { TrashPage } from "./Pages/Trash-Page/trash";
```
Comment (gopalbbharadva) [inline src/App.js:9]: Suggestion : You can make export of pages.

File: src/components/Aside/Aside.jsx [hunk 2]
```diff
@@ -8,47 +8,50 @@ import { AiOutlineLogout } from 'react-icons/ai';
...
 const Aside = () => {
 
     const navigate = useNavigate();
     const  { authDispatch } = useAuth();
+    const { activePage, setActivePage } = useNote();
 
     // user logout click handler
     const logoutClickHandler = (e) => {
         console.log("clicked");
         localStorage.removeItem("token");
```
Comment (gopalbbharadva) [inline src/components/Aside/Aside.jsx]: Remove consoles if not needed

**Review by gopalbbharadva — APPROVED** [review]
Bug: If I press edit and send that item to trash or archives, inputs are still filled with its values, so just clear the inputs if user is editing and trashing the same note.

**Comment by Jay-Gadhiya** [comment]
> Bug: If I press edit and send that item to trash or archives, inputs are still filled with its values, so just clear the inputs if user is editing and trashing the same note.

thanks for finding this bug, I will resolve this.

**Review by aamir74 — APPROVED** [review]
Bug: In archives tab when clicked on trash button of notes it get deleted but unable to find that trashed note in trash section(page), rest is good to go.
code is clean.

**Review by gadgetfather — APPROVED** [review]
- Great use of context
- All dispatch calls are very descriptive
- Functionality is working fine

**Review by jeetbhalani15 — APPROVED** [review]
-Make changes as suggested by @gopalbbharadva.
-UI Look clean.
-All API calls are working fine

**Comment by Jay-Gadhiya** [comment]
> Bug: In archives tab when clicked on trash button of notes it get deleted but unable to find that trashed note in trash section(page), rest is good to go. code is clean.

i will add that functionality.

**Pull request merged**